Index: wrapped/templates/wrapped/home.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!-- templates/wrapped/home.html -->\n{% extends 'wrapped/base.html' %}\n\n{% block content %}\n    <p class=\"text-blue-500\">This is a Django app styled with Tailwind CSS.</p>\n{% endblock %}\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/wrapped/templates/wrapped/home.html b/wrapped/templates/wrapped/home.html
--- a/wrapped/templates/wrapped/home.html	(revision 4f3a521576a1295bed54ec37b599b563ee4484e1)
+++ b/wrapped/templates/wrapped/home.html	(date 1730397585526)
@@ -2,5 +2,5 @@
 {% extends 'wrapped/base.html' %}
 
 {% block content %}
-    <p class="text-blue-500">This is a Django app styled with Tailwind CSS.</p>
+    <p class="text-blue-500">Welcome to Spotify Wrapped.</p>
 {% endblock %}
Index: wrapped/templates/wrapped/base.html
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><!DOCTYPE html>\n<html lang=\"en\">\n<head>\n    <meta charset=\"UTF-8\">\n    <meta name=\"viewport\" content=\"width=device-width, initial-scale=1.0\">\n    <title>My Django Project</title>\n    <!-- Tailwind CSS CDN -->\n    <link href=\"https://cdn.jsdelivr.net/npm/tailwindcss@2.2.19/dist/tailwind.min.css\" rel=\"stylesheet\">\n</head>\n<body>\n    <div class=\"container mx-auto\">\n        <h1 class=\"text-3xl font-bold\">Hello, Django + Tailwind!</h1>\n        {% block content %}{% endblock %}\n    </div>\n</body>\n</html>\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/wrapped/templates/wrapped/base.html b/wrapped/templates/wrapped/base.html
--- a/wrapped/templates/wrapped/base.html	(revision 4f3a521576a1295bed54ec37b599b563ee4484e1)
+++ b/wrapped/templates/wrapped/base.html	(date 1730397619996)
@@ -9,7 +9,7 @@
 </head>
 <body>
     <div class="container mx-auto">
-        <h1 class="text-3xl font-bold">Hello, Django + Tailwind!</h1>
+        <h1 class="text-3xl font-bold">Spotify Wrapped</h1>
         {% block content %}{% endblock %}
     </div>
 </body>
